src/components/nodes-table/nodes-table.js

```diff
@@ -10,9 +10,10 @@ export default {
     filterOptions
   },
   data: () => ({
-    nodeTableItems: 10,
+    nodeTableItems: undefined,
```

Comment:
This is set to undefined for 2 reasons:
1. Default values should be set only once (see mounted())
2. This ensures that the default value is explicitely set and triggers the watch function to ensure there are always valid settings stored in the local store.